src/backend/cpu/memory.hpp

```diff
@@ -9,7 +9,7 @@
 #pragma once
 
 #include <af/defines.h>
-#include <common/MemoryManager.hpp>
+#include <MemoryManager.hpp>
```

Comment:
I'd rather keep `common/MemoryManager.hpp`. Removing this may cause conflicts in the future.